torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -1812,7 +1812,8 @@ std::vector<std::shared_ptr<NCCLComm>>& ProcessGroupNCCL::getNCCLComm(
     const std::vector<at::Device>& devices,
     OpType opType,
     int p2pRank,
-    bool isSendRecvSelf) {
+    bool isSendRecvSelf,
+    bool eagerMode) {
```

Comment:
this feels like it should be earlier in the list of parameters, maybe before or after `opType`; can we change this API since it's internal?  it also would let us make it explicit rather than having a default value which seems nicer

Reply:
After removing the eager mode branch in the new version, we wont need this parameter anymore